dotCMS/src/main/java/com/dotcms/graphql/InterfaceType.java

```diff
@@ -110,32 +79,55 @@
         interfaceTypes.put("CONTENT", createInterfaceType(CONTENT_INTERFACE_NAME, contentFields, new ContentResolver()));
 
         final Map<String, TypeFetcher> fileAssetFields = new HashMap<>(contentFields);
+        addBaseTypeFields(fileAssetFields, ImmutableFileAssetContentType.builder().name("dummy")
```

Comment:
Codacy found an issue: [The String literal "dummy" appears 7 times in this file; the first occurrence is on line 81](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5939440)